pvlib/iotools/panond.py

```diff
@@ -0,0 +1,197 @@
+"""
+Get .PAN or .OND file data into a nested dictionary.
+"""
+
+import io
+
+
+def num_type(value):
+    """
+    Determine if a value is float, int or a string
+    """
+    if '.' in value:
+        try:  # Detect float
+            value_out = float(value)
+            return value_out
+
+        except ValueError:  # Otherwise leave as string
+            value_out = value
+            return value_out
+
+    else:
+
+        try:  # Detect int
+            value_out = int(value)
+            return value_out
+
+        except ValueError:  # Otherwise leave as string
+            value_out = value
+            return value_out
+
+
+def element_type(element):
```

Comment:
```suggestion
def _element_type(element):
```
Adding an underscore at the beginning of the name makes it a "private" function, which I think is what we want here.  Same for `num_type`.

Reply:
Thanks! I didn't know about private functions, despite seeing the leading underscore before.